complete_streaming_function.py

```
@@ -0,0 +1,394 @@
+                if target_persona:
+                    # Use LangChain persona agent for response
+                    if orchestrator.langchain_enabled:
+                        try:
+                            ai_response = await orchestrator.chat_with_persona_langchain(
+                                message=request.message,
+                                persona_id=persona_id,
+                                scene_id=correct_scene_id
+                            )
+                            persona_name = target_persona['identity']['name']
+                            persona_id = target_persona.get('db_id')
+                        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)